DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ScheduleCell.java

```diff
@@ -161,6 +182,24 @@ private String convertSpeakersToString(List<TalkSpeaker> speakers) {
         }
         return "";
     }
+    
+     private PseudoClass fetchPseudoClassForTrack(String trackId) {
+        String pseudoClass = trackPseudoClassMap.get(trackId);
+        if (pseudoClass == null) {
+            if (index > pseudoClasses.size() - 1) {
+                index = 0; // exhausted all colors, re-use
+            }
+            pseudoClass = pseudoClasses.get(index++);
+            trackPseudoClassMap.put(trackId, pseudoClass);
+        }
+        return PseudoClass.getPseudoClass(pseudoClass);
```

Comment:
Instead of creating the pseudo class all the time (which in turn just gets it from an internal map), don't you think it could be better to add them to our map `trackPseudoClassMap`, using  `Map<String, PseudoClass>`?